tools/user_management/lib/group.rb

```diff
@@ -0,0 +1,54 @@
+require 'json'
+
+require_relative 'uaa_resource'
+
+class Group < UAAResource
+  attr_reader :name, :users
+
+  def initialize(name, users)
+    @name = name
+    @users = users
+  end
+
+  def remove_unexpected_members(uaa_client)
+    get_members(uaa_client).each do |member|
+      member_guid = member['value']
+      unless @users.any? { |user| user.guid == member_guid || member['origin'] == 'uaa' }
```

Comment:
What is this line of code trying to do? Preserve the `admin` user?

Reply:
Yeah... The member object only contains three fields...

```json
{
  "value": "00000000-0000-0000-0000-000000000000",
  "type": "USER",
  "origin": "uaa" // or `google`
}
```

I thought that If we want to use google as our source of truth for that, we can essentially ignore users with origin of `uaa`, which coincidentally include the `admin` user. And we don't want to loose that...

Reply:
I ended up rewriting things to use the list members endpoint which does include username. Deploying to `miki` this morning and hopefully ship it after :)